docs/workforce-management-framework/topics/con-supported-modules.adoc

```diff
@@ -0,0 +1,12 @@
+= {WFM-RC-NameLong} Supported Modules
+
+An up to date list of modules can be found at https://www.npmjs.com/org/raincatcher/. The supported list is below
```

Comment:
@austincunningham Please extend this by at least providing once sencence for module. 
We can also group them together in some form of paragraps for:

- Integration modules
- WFM modules

We may also mention angular js module that we do not document, but still have in our application.